src/containers/namespace-list/namespace-list.tsx

```diff
@@ -99,6 +100,10 @@ export class NamespaceList extends React.Component<IProps, IState> {
   }
 
   render() {
+    if (this.state.redirect) {
+      return <Redirect to={this.state.redirect} />;
```

Comment:
:+1:, but also https://github.com/ansible/ansible-hub-ui/pull/859#issuecomment-910920191

```suggestion
      return <Redirect push to={this.state.redirect} />;
```

Reply:
Ok.